src/Avalonia.X11/X11Screens.cs

```diff
@@ -82,6 +82,39 @@ private void OnEvent(XEvent ev)
                     _cache = null;
             }
 
+            private unsafe Size? GetPhysicalMonitorSizeFromEDID(IntPtr rrOutput)
+            {
+                if(rrOutput == IntPtr.Zero)
+                    return null;
+                var output = Marshal.ReadInt64(rrOutput);
```

Comment:
Are you sure that it's 64 bit on 32-bit ARM too? Might be better to use `ReadIntPtr`

Reply:
Actually, `_XRRMonitorInfo` is defined like this:
```cs
typedef struct _XRRMonitorInfo {
    Atom name;
    Bool primary;
    Bool automatic;
    int noutput;
    int x;
    int y;
    int width;
    int height;
    int mwidth;
    int mheight;
    RROutput *outputs;
} XRRMonitorInfo;
```
and `RROutput` is a `XID`, so we should make the `Outputs` property to be `IntPtr*`